### DISCUSSION TITLE
Is it possible to add animation (ripple/pulsing) to the coach mark cutout path?

### DISCUSSION THREAD

**Discussion by gowripriya1999** [ephread/Instructions#293] [Q&A]
Is it possible to add animation (ripple/pulsing) to the coach mark cutout path?

https://user-images.githubusercontent.com/51815366/186852528-744d76a5-c283-46e9-8f4b-3b49ba776597.mov

**Comment by ephread** [comment]
This effect is unfortunately not supported by Instructions.
